poller/README.md

```diff
@@ -138,6 +142,23 @@ configuration and specify name (`high_priority_cpu`, `rolling_24_hr` and
 `storage`) of the metric to be changed and desired `regional_threshold` or
 `multi_regional_threshold` for your Cloud Spanner instance.
 
+### Margins
+
+A margin defines an upper and a lower limit around the threshold. An autoscaling
+event will be triggered only if the metric value falls above the upper limit,
+or below the lower limit.
+
+The objective of this parameter is to avoid autoscaling evnts being triggered
+for small workload fluctuations around the threshold. The threshold and metric
```

Comment:
, thus creating a smoothing effect in autoscaler actions.

Something like that?

Reply:
added suggestion